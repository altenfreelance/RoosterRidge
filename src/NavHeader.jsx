import React, { Component } from "react";
import { NavLink } from "react-router-dom";

export default class NavHeader extends Component {
  render() {
    return (
      <header className="sticky-top masthead rounded px-3 py-2">
        <div className="inner">
          <h3 className="masthead-brand">Rooster Ridge</h3>
          <nav className="nav nav-masthead justify-content-center">
            <NavLink to="/home" activeClassName="active" replace>
              Home
            </NavLink>
            <NavLink to="/shows" activeClassName="active" replace>
              Shows
            </NavLink>
            <NavLink to="/members" activeClassName="active" replace>
              Members
            </NavLink>
            <NavLink to="/contact" activeClassName="active" replace>
              Contact
            </NavLink>
            <NavLink to="/music" activeClassName="active" replace>
              Music
            </NavLink>
          </nav>
        </div>
      </header>
    );
  }
}
