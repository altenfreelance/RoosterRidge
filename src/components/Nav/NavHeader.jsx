import React, { Component } from "react";
import { NavLink } from "react-router-dom";

export default class NavHeader extends Component {
  render() {
    return (
      <header className="sticky-top masthead rounded px-3 py-2">
        <div className="inner">
          {/* <h3 style={{fontFamily: 'serif', textTransform: 'title'}} className="masthead-brand">Rooster Ridge</h3> */}
          <nav className="nav nav-masthead justify-content-center">
            <NavLink to="/about" activeClassName="active" replace>
              About
            </NavLink>
            <NavLink to="/shows" activeClassName="active" replace>
              Shows
            </NavLink>
            <NavLink to="/members" activeClassName="active" replace>
              Members
            </NavLink>
            <NavLink to="/contact" activeClassName="active" replace>
              Contact
            </NavLink>
            <NavLink to="/music" activeClassName="active" replace>
              Music
            </NavLink>
          </nav>
        </div>
      </header>
    );
  }
}
