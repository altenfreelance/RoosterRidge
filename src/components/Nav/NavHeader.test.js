import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import NavHeader from './NavHeader';
import {
    BrowserRouter as Router
} from "react-router-dom";

it('renders without crashing', () => {
const div = document.createElement('div');

ReactDOM.render(<Router><NavHeader /></Router>, div);

ReactDOM.unmountComponentAtNode(div);
});
