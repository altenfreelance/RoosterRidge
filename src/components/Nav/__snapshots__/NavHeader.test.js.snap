// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders (matches snapshot)  1`] = `
<DocumentFragment>
  <header
    class="sticky-top masthead rounded px-3 py-2"
  >
    <div
      class="inner"
    >
      <h3
        class="masthead-brand"
        style="font-family: serif; text-transform: title;"
      >
        Rooster Ridge
      </h3>
      <nav
        class="nav nav-masthead justify-content-center"
      >
        <a
          href="/home"
        >
          Home
        </a>
        <a
          href="/shows"
        >
          Shows
        </a>
        <a
          href="/members"
        >
          Members
        </a>
        <a
          href="/contact"
        >
          Contact
        </a>
        <a
          href="/music"
        >
          Music
        </a>
      </nav>
    </div>
  </header>
</DocumentFragment>
`;
